PC/pyconfig.h

```diff
@@ -690,4 +690,7 @@ Py_NO_ENABLE_SHARED to find out.  Also support MS_NO_COREDLL for b/w compat */
 /* Define to 1 if you have the `erfc' function. */
 #define HAVE_ERFC 1
 
+/* framework name */
+#define PYTHONFRAMEWORK ""
+
```

Comment:
Travis CI failed:

Fixing C file whitespace ... 1 file:

  PC/pyconfig.h

Fixing docs whitespace ... 0 files

Please fix the 1 file(s) with whitespace issues

(on UNIX you can run `make patchcheck` to make the fixes)